samples/training/s3_sample_data_creator.py

```diff
@@ -0,0 +1,51 @@
+import shap
```

Comment:
afaik, this is not the right script for creating data, examples use mnist dataset